web/src/api/ipfix.ts

```diff
@@ -62,22 +66,30 @@ export interface Fields {
   DstK8S_HostName?: string;
   /** L4 protocol */
   Proto: number;
+  /** Network interface */
+  Interface?: string;
   /** Number of packets in this flow */
   Packets: number;
+  /** In conversation tracking, A to B packets counter */
   Packets_AB?: number;
+  /** In conversation tracking, B to A packets counter */
   Packets_BA?: number;
   /** Number of bytes in this flow */
   Bytes: number;
+  /** In conversation tracking, A to B bytes counter */
   Bytes_AB?: number;
+  /** In conversation tracking, B to A bytes counter */
   Bytes_BA?: number;
   /** Start timestamp of this flow, in milliseconds */
   TimeFlowStartMs: number;
   /** End timestamp of this flow, in milliseconds */
   TimeFlowEndMs: number;
   /** Timestamp when this flow was received and processed by the flow collector, in seconds */
   TimeReceived: number;
+  /** In conversation tracking, the conversation identifier */
   _HashId?: string;
+  /** In conversation tracking, flag identifying the first flow */
   _IsFirst?: string;
+  /** In conversation tracking, a counter of flow logs */
```

Comment:
Hey @jpinsonneau @ronensc : is this accurate, this is the number of flows already accumulated in a conversation?

Reply:
yes :+1: good to me